### DISCUSSION TITLE
Set subdirectory as root instead of working directory

### DISCUSSION THREAD

**Discussion by josh-aliencode** [DirectoryLister/DirectoryLister#915] [Help & Support]
Instead of using the current working directory.. How would I go about setting a specific subfolder for the root/home directory-lister? I wish to mount another folder from my system there.

For example.. Starting in
/DirectoryLister-3.12.0/files/
instead of
DirectoryLister-3.12.0/

PS.. Great work!! I love this.

**Comment by DeMiro5001** [comment]
Did you manage to get it working? I am looking for the same thing

**Comment by PHLAK** [comment]
To list a sub-folder, install Directory Lister to the sub-folder instead of the parent folder.

**Reply by DeMiro5001** [reply]
Thanks a lot for the super quick reply. That will be the last resort.
Can I add other locations that have the correct permissions and ownership?

**Reply by DeMiro5001** [reply]
It looks like https://github.com/DirectoryLister/DirectoryLister/discussions/736#discussioncomment-1522535
I will test it, tomorrow
